app/views/categories/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if !user_signed_in? %>
+  <div class="start-screen f c">
+    <div class="app-name f">
+      <h1>
+        SpendTracker
+      </h1>
+    </div>
+    <div class="start-buttons f c">
+      <%= link_to "Log in", new_session_path(:user), class: 'login f'%>
+      <%= link_to "Sign up", new_registration_path(:user), class: 'signup f' %>
+    </div>
+  </div>
+<% else %>
+  <div class="top-bar f r">
+    <%= link_to "←", destroy_user_session_path, method: :delete, data: { confirm: "Are you sure you want to sign out?" }, class:"back-button f r" %>
+    <h2 id="title">Categories</h2>
+    <%= link_to "New", new_category_path, class: 'topbar-button' %>
+  </div>
+  <%= render 'shared/category_info', categories: @categories, category_totals: @category_totals %>
```

Comment:
Dear @evillaz, Your Implementation is really interesting and I like your app. But please consider the following minor changes. 
- [ ] For the home page we need to have a button at the bottom, I can see you have a button at the top (`new`), Please move it to the bottom, rename it to `Add new category`, and apply the appropriate button style. This makes it convenient for the end user.
- [ ] This also goes for the "add a new transaction" button, as you can see from the original design the buttons are at the bottom and has their own styes.
### Original design 
![Screen Shot 2023-07-22 at 3 29 57 PM](https://github.com/evillaz/spendtracker/assets/69077061/aea46f3f-2939-4b3a-a776-002aff051d77)
### Your design 
![Screen Shot 2023-07-22 at 3 33 08 PM](https://github.com/evillaz/spendtracker/assets/69077061/bf4e0715-57f4-4293-96cb-09079aeb24ae)